.github/workflows/rendersite.yml

```diff
@@ -4,14 +4,21 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v4
         with:
           java-version: '11'
           distribution: 'adopt'
-          cache: gradle
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          # Publish a Gradle build scan so we can see all the build logs,
+          # a complete task timeline, test outputs, and the resolved dependencies of the build.
+          build-scan-publish: true
```

Comment:
We have a Develocity (former Gradle Enterprise) instance (configured in settings.gradle. With that, the only item needed here is develocity-access-key: ${{ secrets.GRADLE_ENTERPRISE_ACCESS_KEY }}.